perllib/FixMyStreet/Cobrand/UKCouncils.pm

```diff
@@ -267,6 +267,26 @@ sub prefill_report_fields_for_inspector { 1 }
 
 sub social_auth_disabled { 1 }
 
+
+=head2 response_template_body_ids
```

Comment:
This is okay, but looking at it like this - is there a reason why we use body areas at all, and not just the body (as this override does) everywhere?

Reply:
That's a good question. Might have been something to with districts for Oxfordshire? Either way, don't think it's in use any more. Have removed the area_ids reference in 1108af9070fe520f4f40f6e84e299fde8d200611 - does that look OK?

Reply:
Yeah! No harm in leaving the area_id flow in, I guess, though happy to remove it as well (and ha, I guess doesn't need params any more if it's only ever taking a body ID argument)